modules/glusterfs/storagedglusterfsinfo.c

```diff
@@ -0,0 +1,230 @@
+/* -*- mode: C; c-file-style: "gnu"; indent-tabs-mode: nil; -*-
+ *
+ * Copyright (C) 2015 Samikshan Bairagya <sbairagy@redhat.com>
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+ */
+
+#include <libxml/parser.h>
+#include <glib.h>
+#include <stdlib.h>
+#include <string.h>
+
+#include <src/storagedlogging.h>
+
+#include "storagedglusterfsinfo.h"
+
+GVariantBuilder *builder, *b;
+
+static void
+add_volume_name_to_list(xmlNode *cur)
+{
+  cur = cur->children;
+  while (cur != NULL) {
```

Comment:
Please, consider GNU coding style (braces at the new line, indented by 2 spaces).